test/e2e_node/mirror_pod_test.go

```diff
@@ -272,6 +272,90 @@ var _ = SIGDescribe("MirrorPod", func() {
 		})
 
 	})
+})
+
+var _ = SIGDescribe("MirrorPod (Pod Generation)", func() {
```

Comment:
intentionally didn't put the `observedGeneration` feature gate here, because the behavior doesn't change whether the feature gate is on or off (it doesn't get set for mirror pods regardless)